tables/automl/automl_tables_dataset.py

```diff
@@ -288,7 +308,7 @@ def import_data(project_id, compute_region, dataset_display_name, path):
     client = automl.TablesClient(project=project_id, region=compute_region)
 
     response = None
-    if path.startswith('bq'):
+    if path.startswith("bq"):
```

Comment:
Again, I prefer single-quotes over double quotes and this is what's in our Python style guide.